Gems/ROS2/Code/Source/LidarTemplate.h

```diff
@@ -70,19 +71,30 @@ namespace ROS2
         {
             directions.clear();
```

Comment:
Better would be just to return AZStd::vector; it will be perf-equivalent due to the RVO / copy elision.

Reply:
Right, also the interface would be cleaner a bit. I'll change it soon.